javascript/index.js

```diff
@@ -1,30 +1,57 @@
 const chronometer = new Chronometer();
-
 // get the buttons:
+
 const btnLeft = document.getElementById('btnLeft');
-const btnRight = document.getElementById('btnRight');
+btnLeft.onclick = function () {
+  if (btnLeft.className === "btn start") {
+    chronometer.startClick(printTime);
+    // printTime as a callback
+    // two functions printMinutes and printSeconds
+  } else if (btnLeft.className === "btn stop") {
+    chronometer.stopClick()
```

Comment:
Just like before, we want our buttons to change accordingly using some DOM manipulation
```
btnLeft.className = 'btn start';
btnLeft.textContent = 'START';
btnRight.className = 'btn reset';
btnRight.textContent = 'RESET';